core/trino-main/src/test/java/io/trino/operator/join/BenchmarkHashBuildAndJoinOperators.java

```diff
@@ -78,10 +80,10 @@
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.openjdk.jmh.annotations.Mode.AverageTime;
-import static org.openjdk.jmh.annotations.Scope.Thread;
 
 @SuppressWarnings("MethodMayBeStatic")
-@State(Thread)
+@State(Scope.Benchmark)
+@Threads(Threads.MAX)
```

Comment:
I understand there is no easy way to parametrize this?

Reply:
I don't think so